Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -184,6 +184,30 @@ def SwapKeepAndDrop(l):
                                   'keep DetIds_hltSiStripRawToDigi_*_*'
                               ])
 
+#
+# HLTSCOUT Data Tier definition
+#
+#
+HLTSCOUTEventContent = cms.PSet(
+    outputCommands = cms.untracked.vstring('drop *'),
+    splitLevel = cms.untracked.int32(0),
+    compressionAlgorithm=cms.untracked.string("LZMA"),
+    compressionLevel=cms.untracked.int32(4)
+)
+HLTSCOUTEventContent.outputCommands.extend(HLTriggerRAW.outputCommands)
```

Comment:
we've been discussing internally within TSG that this could be just [`HLTScouting`](https://github.com/cms-sw/cmssw/blob/d389c1c21ce701feb341cd00236d5b432160d2b5/HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py#L412) or even better we could add a new `HLTriggerHLTSCOUT` to `HLTrigger_EventContent_cff.py` replacing `HLTScouting` (same `keep` instructions but adding a ` 'drop *_hlt*_*_*'` at the beginning). This would need to be done internally by TSG and can go as a refinement after this PR is integrated.